doc/content/devices/best-practices/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Best Practices"
+description: ""
+weight:
+---
+
+This section contains best practices for building connected devices which use {{% tts %}} as a Network Server.
+
+<!--more-->
+
+LoRaWAN devices should always comply to the [LoRaWAN specification](https://www.lora-alliance.org/lorawan-for-developers), with special regard to minimizing join requests and duty cycle limitations.
+
+## Minimize Join Requests
```

Comment:
I don't really like this heading as a "best practice". Joins are not bad, only unnecessary joins are.